File: GameRanksClient/src/app/models/serverStructs/page/DeveloperPageStruct.ts
import {DeveloperStruct} from "../element/DeveloperStruct";
import {StatisticStruct} from "../element/StatisticStruct";

export class DeveloperPageStruct{
	developerStruct: DeveloperStruct;
	statisticStruct: StatisticStruct;

	constructor(developerStruct: DeveloperStruct, statisticStruct: StatisticStruct){
		this.developerStruct = developerStruct;
		this.statisticStruct = statisticStruct;
	}
}

File: GameRanks/src/main/java/GameRanks/GameRanks/clientStruct/page/DeveloperPageStruct.java
package GameRanks.GameRanks.clientStruct.page;

import GameRanks.GameRanks.clientStruct.element.DeveloperStruct;
import GameRanks.GameRanks.serverStruct.StatisticStruct;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class DeveloperPageStruct {
    private DeveloperStruct developerStruct;
    private StatisticStruct statisticStruct;
}
